app/src/main/res/layout/activity_partners.xml

```diff
@@ -25,23 +26,16 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
         android:clickable="true"
         android:focusable="true"
         android:focusableInTouchMode="true"
+        android:orientation="vertical"
+        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
Why is this necessary?

Reply:
You mean the `LinearLayout`? I answered the question already above?

Reply:
I mean `app:layout_constraintEnd_toEndOf="parent"`

Reply:
You are right, this is not necessary. I going to remove it.